src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
You already have a parameterized constructor for the `Ball` creation. The whole method could be a one-liner:
  `return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));`